test/parallel/test-assert.js

```diff
@@ -277,8 +278,8 @@ test('assert.throws()', () => {
       code: 'ERR_ASSERTION',
       name: 'AssertionError',
       message: 'Expected "actual" to be reference-equal to "expected":\n' +
-              '+ actual - expected\n\n' +
-              '+ [Error: foo]\n- [Error: foobar]'
+        '\n' +
+        '[Error: foo] !== [Error: foobar]\n'
```

Comment:
Single lines could be handled differently by applying colors to the output (when detected as being available, otherwise I would fallback to the former output).

That way it would even be easier to read (in most cases, we could also have an upper bound of difference percentage / different parts) and less output.

Reply:
interesting idea! Do you mind sharing an example of what you have in mind?  
But also keep in mind that, as part of this PR, I wanted to standardize more the code, removing weird exceptions where the output might be different based on, for example, terminal size. Are you sure we want to go down that route? That's a genuine question, just wanting feedback 😄

Reply:
Discussed personally. Resolved.